samples/test-resources/data-mysql/src/main/resources/application.yml

```diff
@@ -16,3 +16,7 @@ datasources:
 #    driverClassName: org.postgresql.Driver
 #    schema-generate: CREATE_DROP
 #    dialect: POSTGRES
+test-resources:
```

Comment:
Sorry for interfering, just to ask can you try using test-containers 1.19.4 in this PR, maybe these mysql overrides won't be needed?
https://github.com/testcontainers/testcontainers-java/releases/tag/1.19.4 contains fix for MySQL it seems.

Reply:
Yes, I plan to test that version, but not in that PR, reason being that it's driven by test resources, not the Gradle plugin.

Reply:
Ah, sorry, I assume it is test-resources as soon as I see your name:)